cmd/objbench.go

```diff
@@ -173,13 +192,9 @@ func objbench(ctx *cli.Context) error {
 		var uploadResult = []string{"multi-upload", nspt, nspt}
 		fname := fmt.Sprintf("__multi_upload__test__%d__", time.Now().UnixNano())
 		if err := blob.CompleteUpload("test", "fakeUploadId", []*object.Part{}); err != utils.ENOTSUP {
-			part := make([]byte, 5<<20)
-			total := int(math.Ceil(float64(fsize) / float64(len(part))))
-			rand.Read(part)
-			content := make([][]byte, total)
-			for i := 0; i < total; i++ {
-				content[i] = part
-			}
+			partSize := 5 << 20
+			total := int(math.Ceil(float64(fsize) / float64(partSize)))
+			content := makeData(total, partSize)
```

Comment:
we should create the content until upload/download, otherwise we can't run large tests.

Reply:
The timing starts in the next step, and the time to generate the data is not counted.